b2luigi/batch/processes/lsf.py

```diff
@@ -30,17 +30,23 @@ class LSFProcess(BatchProcess):
     """
     Reference implementation of the batch process for a LSF batch system.
 
-    Additional to the basic batch setup (see :ref:`batch-label`), additional
-    LSF-specific things are:
+    Additional to the basic batch setup (see :ref:`batch-label`), there are
+    LSF-specific settings. These are:
 
-    * the LSF queue can be controlled via the ``queue`` parameter, e.g.
+    * the LSF queue: ``queue``.
+    * the LSF job name: ``job_name``.
+
+    Both can be controlled via the standard settings interface, e.g.
 
       .. code-block:: python
 
         class MyLongTask(b2luigi.Task):
             queue = "l"
+            job_name = "my_long_task"
 
-      The default is the short queue "s".
+      The default queue is the short queue "s". If no job_name is set the task
+      will appear as "{result_dir}/parameter1=value/.../executable_wrapper.sh"
+      when running ``bjobs``.
```

Comment:
I would put the default parameters and variable names in literal blocks. Also in documentation I usually know the notation to use angle brackets for strings to be substituted instead of curly brackets, though the latter are used to python f-strings and templating systems. I'm not sure but I think the angle-brackets notation originates from the [Backus-Naur form](https://en.wikipedia.org/wiki/Backus%E2%80%93Naur_form). My suggestion:


```suggestion
      The default queue is the short queue ``"s"``. If no ``job_name`` is set the task
      will appear as ::

        <result_dir>/parameter1=value/.../executable_wrapper.sh"

      when running ``bjobs``.
```
![](https://user-images.githubusercontent.com/5121824/111201883-8582f080-85c3-11eb-8f41-e186a4808543.png)